src/views/Home/Home.vue

```diff
@@ -6,20 +6,58 @@
       </div>
       <p>S’Mail is sort of an email client application that enable configuring one or more email addresses to receive, read, compose and send emails from one user to another.</p>
       <div class="btnlist">
-        <router-link to="" class="btn btn-outline-light">About Us</router-link>
+        <button class="btn btn-outline-light" v-b-modal.showus>About Us</button>
```

Comment:
ini harusnya pake `v-on:click="......"` we karna kita mw ngejalanin function pas onclick